### PR TITLE
feat(enhancer-version): allow selecting a specific enhancer version in standalone

### PR DESCRIPTION
**Pull request by chicharr**
Fixes: #203 

Allow users of standalone distribution to provide a specific enhancer version, and a md5 hash

### CODE REVIEW

**Comment by cziegeler** [comment]
@chicharr Looks good to me, but what about the 404 script?

**Review by kptdobe — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ export function sampleRUM(checkpoint, data) {
   const timeShift = () => (window.performance ? window.performance.now() : Date.now() - window.hlx.rum.firstReadTime);
   try {
     window.hlx = window.hlx || {};
-    sampleRUM.enhance = () => {};
+    sampleRUM.enhance = () => { };
```
Comment (kptdobe) [inline src/index.js:18]:
```suggestion
    sampleRUM.enhance = () => {};
```

**Comment by cziegeler** [comment]
@chicharr As mentioned, the PR looks good, but the same needs to be applied to the standalone 404 script

**Comment by chicharr** [comment]
> @chicharr As mentioned, the PR looks good, but the same needs to be applied to the standalone 404 script

@cziegeler @kptdobe  @trieloff PR is updated with the changes in 404.js

**Review by kptdobe — APPROVED** [review]

**Review by trieloff — APPROVED** [review]

**Review by cziegeler — APPROVED** [review]

**Pull request merged**